includes/integration.drush.inc

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file
+ * Contains Drush-related hook implementations.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function integration_drush_command() {
+  $items = array();
+  $items['integration-export-document'] = array(
```

Comment:
yeah let's use `integration-export` and refine it via arguments, it will give us a consistent base to build upon. Thanks!